linux/vcpkg_config_manager/vcpkg.json

```diff
@@ -0,0 +1,14 @@
+{
+    "name": "boinc-client",
```

Comment:
This is not boinc-client but boinc-manager

Reply:
You are correct. I will change that.